entity-framework/efcore-and-ef6/index.md

```diff
@@ -1,77 +1,74 @@
+The following tables compare the features available in EF Core and EF6. This is a high-level comparison and doesn't list every feature or explain differences between the same feature in different EF versions.
 
 The EF Core column indicates the product version in which the feature first appeared.
 
 ### Creating a model
 
-| **Feature**                                           | **EF 6** | **EF Core**                           |
+| **Feature**                                           | **EF6.4**| **EF Core**                           |
 |:------------------------------------------------------|:---------|:--------------------------------------|
 | Basic class mapping                                   | Yes      | 1.0                                   |
 | Constructors with parameters                          |          | 2.1                                   |
 | Property value conversions                            |          | 2.1                                   |
 | Mapped types with no keys                             |          | 2.1                                   |
 | Conventions                                           | Yes      | 1.0                                   |
-| Custom conventions                                    | Yes      | 1.0 (partial)                         |
+| Custom conventions                                    | Yes      | 1.0 (partial; [#214](https://github.com/dotnet/efcore/issues/214)) |
 | Data annotations                                      | Yes      | 1.0                                   |
 | Fluent API                                            | Yes      | 1.0                                   |
 | Inheritance: Table per hierarchy (TPH)                | Yes      | 1.0                                   |
-| Inheritance: Table per type (TPT)                     | Yes      |                                       |
-| Inheritance: Table per concrete class (TPC)           | Yes      |                                       |
+| Inheritance: Table per type (TPT)                     | Yes      | Planned for 5.0 ([#2266](https://github.com/dotnet/efcore/issues/2266)) |
+| Inheritance: Table per concrete class (TPC)           | Yes      | Stretch for 5.0 ([#3170](https://github.com/dotnet/efcore/issues/3170)) <sup>(1)</sup> |
 | Shadow state properties                               |          | 1.0                                   |
 | Alternate keys                                        |          | 1.0                                   |
-| Many-to-many without join entity                      | Yes      |                                       |
+| Many-to-many navigations                              | Yes      | Planned for 5.0 ([#19003](https://github.com/dotnet/efcore/issues/19003)) |
+| Many-to-many without join entity                      | Yes      | Backlogged ([#1368](https://github.com/dotnet/efcore/issues/1368)) |
```

Comment:
> ~~Backlogged~~ **On the backlog**

Backlogged sounds like we were going to do it but then decided to punt it.

Reply:
Or just "Backlog"